README.md

```diff
@@ -299,12 +266,6 @@ And attach the gzipped log file to your bug report.
 
 Without `WHIPPER_LOGFILE` set, logging messages will go to stderr. `WHIPPER_DEBUG` accepts a string of the [default python logging levels](https://docs.python.org/2/library/logging.html#logging-levels).
 
-### Developing
-
-Pull requests are welcome.
-
-**WARNING:** As whipper is still under heavy development sometimes I will force push (`--force-with-lease`) to the non master branches.
-
 ## Credits
```

Comment:
I sort of want to delete this section, as it's easy to see in the git history who the developers are, and we already have all these names (+ Freso's now) in the copyright section.

Reply:
:+1: